### PR TITLE
Footnote support

### PR DESCRIPTION
**Pull request by michael**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 .DS_Store
 /node_modules/
+data/ams
```
Comment (pkra) [inline .gitignore:3]: Please undo this change.

File: server.js [hunk 2]
```diff
@@ -68,4 +68,5 @@ app.use(app.router);
 http.createServer(app).listen(port, function(){
   console.log("Lens running on port " + port);
   console.log("http://127.0.0.1:"+port+"/");
+  console.log("To open a local example file goto http://127.0.0.1:"+port+"?url=data/ams/bproc10/bproc10.xml");
```
Comment (pkra) [inline server.js:71]: Please undo this change.

Comment (obuchtala) [inline server.js:71]: I will, when you tried to explain your position.

Comment (pkra) [inline server.js:71]: Both of these are specific to a local setup. The latter also carries non-public information.

Comment (obuchtala) [inline server.js:71]: Agree.

Comment (michael) [inline server.js:71]: Fine if we just revert them or you want them to disappear from the commit history entirely? Would need to rewrite the devel branch to do that.

Comment (pkra) [inline server.js:71]: > Fine if we just revert them or you want them to disappear from the commit history entirely?

Just a revert is fine. Thanks.
